lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -2174,177 +2202,11 @@ fn do_test_trampoline_single_hop_receive(success: bool) {
 #[cfg(trampoline)]
 fn test_trampoline_single_hop_receive() {
 	// Simulate a payment of A (0) -> B (1) -> C(Trampoline (blinded intro)) (2)
-	do_test_trampoline_single_hop_receive(true);
-
-	// Simulate a payment failure of A (0) -> B (1) -> C(Trampoline (blinded forward)) (2)
-	do_test_trampoline_single_hop_receive(false);
-}
+	do_test_trampoline_single_hop_receive(TrampolineSingleHopReceiveCheck::SuccessfulBlindedReceive);
```

Comment:
Is there currently a test covering the forwarding operation of trampoline node? It's not yet implemented, but to assert that nothing weird happens if we receive a forward?

Reply:
Especially without the cfg flag.